src/test/java/com/bouncestorage/chaoshttpproxy/ChaosHttpProxyTest.java

```diff
@@ -68,10 +75,27 @@
     private HttpBin httpBin;
     private HttpClient client;
 
+    private ChaosConfig chaosConfig;
+
     @Before
     public void setUp() throws Exception {
-        proxy = new ChaosHttpProxy(proxyEndpoint,
-                Suppliers.ofInstance(Failure.SUCCESS));
+        chaosConfig = new ChaosConfig() {
```

Comment:
Better to exercise the real `DefaultChaosConfig` instead of this custom implementation?